components/board/status.go

```diff
@@ -17,8 +17,8 @@ func CreateStatus(ctx context.Context, b Board, extra map[string]interface{}) (*
 	if names := b.AnalogNames(); len(names) != 0 {
 		status.Analogs = make(map[string]*commonpb.AnalogStatus, len(names))
 		for _, name := range names {
-			x, ok := b.AnalogByName(name)
-			if !ok {
+			x, err := b.AnalogByName(name)
+			if err != nil {
 				return nil, fmt.Errorf("analog %q not found", name)
```

Comment:
you can just return `err` here since it will return the same error message, that way we don't have to rewrite the error message every time.